dogpile/util/langhelpers.py

```diff
@@ -2,6 +2,8 @@
 import re
 import threading
 
+import stevedore
```

Comment:
**mike bayer** (zzzeek) wrote:

is there any memory advantage to importing stevedore inside the conditional below?  basically if someone uses dogpile with the provided backends then the "self._mgr" part is not reached.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/2091

Reply:
**Doug Hellmann** (dhellmann) wrote:

Maybe a small one. It's not a very big package. I assumed pkg_resources was imported there because of the scanning side-effect, but if it was about memory then I could move this import.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/2091

Reply:
**mike bayer** (zzzeek) wrote:

it probably was about memory but it doesn't matter.  this just needs a changelog file in docs/build/unreleased and I can merge it.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/2091

Reply:
I've added a changelog file, let me know if the content looks OK.

You might also be interested in https://docs.openstack.org/reno/latest/ since it lets you have similar collision-free release notes without having to manually build a changelog file (it integrates with sphinx).